README.md

```diff
@@ -22,77 +22,96 @@ create a plethora of workload characteristics that rely on a common storage mode
 spack env activate -p .
 spack install -v
 ```
+ :information_source: Installation can take > 30 minutes.
 
-The installation may take some time (> 30 minutes) to finish.
-
-### Build ChronoLog
+### Building ChronoLog
 
-**Please make sure all the building is carried out in the activated Spack environment.** Otherwise, CMake will not able
-to find the dependencies.
-
-Two executables, `chronovisor_server` and `chrono_keeper`, will be built for ChronoVisor and ChronoKeeper, respectively.
-Multiple client test cases will be built in the `test/integration/Client/` directory. An additional command line client
-admin tool `client_admin` will be built in the `Client/ChronoAdmin` directory as a workload generator.
+ :exclamation: Ensure all build steps are performed within the activated Spack environment to allow CMake to locate necessary dependencies:
 
 ```
+// Build the environment
 cd ChronoLog
 git switch develop
-mkdir build
-cd build
-cmake ..
+mkdir build && cd build
+
+// Build the project 
+cmkae ..
 make all
 ```
 
-### Install ChronoLog
+Building ChronoLog generates the following executables:
+
+- **Servers:** `chronovisor_server` for ChronoVisor and `chrono_keeper` for ChronoKeeper.
+- **Client Test Cases:** Located in `test/integration/Client/`.
+- **Admin Tool:** `client_admin` in `Client/ChronoAdmin` serves as a workload generator.
 
-You can run `make install` in the build directory to install all generated executables along with their dependencies to
-the install directory (`~/chronolog` by default).
+### Installing ChronoLog
 
-### Configuration files
+From the build directory, execute
 
-All ChronoLog executables need a configuration file to run properly. The template file can be found
-in `default_conf.json.in`. You can modify it for your own preferences. The default installing process will copy and
-rename `default_conf.json.in` into `conf` under the install directory. You can pass it to the executables via command
-line argument `--config default_conf.json`.
+```
+make install
+```
 
-ChronoLog will support sockets/TCP/verbs protocols using ofi transport. You can run command `margo-info` to check which
-transports and protocols are supported on your system.
+to install executables and dependencies into the default install directory (**`~/chronolog`**).
 
-------
+## Configuration files
+- **Default Configuration:** ChronoLog executables require a configuration file (`default_conf.json.in`). Modify this template according to your preferences.
+- **Installation Process:** The default installation copies and renames `default_conf.json.in` to `conf` in the install directory.
+- **Using Configuration:** Pass the configuration file to executables with `-config default_conf.json`.
 
+
+
+------
 # Coming soon ...
 
-For more details about the ChronoLog project, please visit our website http://chronolog.dev.
+For more details about the ChronoLog project, please visit our [website](https://grc.iit.edu/research/projects/chronolog)
```

Comment:
switch to the old one. We bought this domain and have been renewing it since the beginning of this project. We surely need to point it to the new grc URL though.

Reply:
Older one says deprecated and shows a link to the grc.iit.edu . I think we should change that then. Otherwise it looks like documentation is not up to date